models/src/main/java/com/vimeo/networking2/Subscription.kt

```diff
@@ -0,0 +1,17 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Information about the user's subscription.
+ */
+@JsonClass(generateAdapter = true)
+data class Subscription(
+
+    @Json(name = "renewal")
+    val renewal: SubscriptionRenewal,
```

Comment:
properties could have Kdocs (or they could be written as param docs)

Reply:
should have a default value of `null`

Reply:
- Good point on using param docs. The other Kotlin DTOs use inline kdocs but these could all be changed to `@param` docs. We should probably stay consistent. I'll add as inline docs for now and will cc @msya on this one to consider future direction of these docs.

- All the fields that don't have a default of null are not nullable (per the API specs). Are we treating **everything** as nullable regardless?

Reply:
Elsewhere in the library, we have always defaulted everything to nullable. The reason is that providing a field filter that excludes a property marked as non null will result in a parsing error. Considering that field filters can be injected by external consumers of the library, properties will need to be nullable. If we control the field filters internally in the library for a certain DTO then we could theoretically make the properties non null. I'd like @msya to weigh in on his thoughts as designer of the DTOs.